hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/ITestS3AFSMainOperations.java

```diff
@@ -58,6 +60,15 @@ public void tearDown() throws Exception {
     }
   }
 
+  @Test
+  @Override
+  public void testWDAbsolute() throws IOException {
+    Path absoluteDir = getTestRootPath(fSys, "test/existingDir");
```

Comment:
could you please explain this test a bit more.

Reply:
override of the superclass. asserts that working directories are absolute